<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoCanvasTest</class>
 <widget class="QWidget" name="VideoCanvasTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSlider" name="progressSlider">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadButton">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="danmakuButton">
       <property name="text">
        <string>Load Danmaku</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="puaseButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Puase</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
